include/glow/Graph/Graph.h

```diff
@@ -261,10 +270,78 @@ class Function final : public Named {
 
   const Module *getParent() const { return parent_; }
 
+  /// Logs the node creation with a list of input nodes.
```

Comment:
Notice that this file is included in all of our compilation unit. This means that the three long methods that you've added here will be parsed and compiled for each one of the files in the project and will slow down the compilation process.

Reply:
moved to Log.cpp